api/src/services/xml_generation/constants.py

```diff
@@ -0,0 +1,15 @@
+"""Constants for XML generation services.
+
+This module contains shared constants, regex patterns, and other values
+used across the XML generation functionality.
+"""
+
+# Currency validation regex from SF-424A form schema
+# Allows: empty string, integers, decimals with exactly 2 places, decimals starting with dot
+CURRENCY_REGEX = r"^\d*([.]\d{2})?$"
```

Comment:
Do we need the non-negative currency regex? I think each of the forms allows negatives.